### PR TITLE
Add static theme option and theming setup page for Android 

### PR DESCRIPTION
**Pull request by ghost**
He was once a thug from around the way

### CODE REVIEW

**Comment by ghost** [comment]
If you want to, you can make orange the default theme, i would reccomend doing so

File: src/android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -505,13 +508,19 @@
...
+    <!-- System theme -->
     <string name="change_theme_mode">Change Theme Mode</string>
     <string name="theme_mode_follow_system">Follow System</string>
     <string name="theme_mode_light">Light</string>
     <string name="theme_mode_dark">Dark</string>
+
+    <!-- Material You theme -->
     <string name="material_you">Material You</string>
-    <string name="material_you_description">Use system colors across the app</string>
+    <string name="material_you_description">Use system colors across the app (Will override (static) theme color option)</string>
```
Comment (Gamer64ytb) [inline src/android/app/src/main/res/values/strings.xml]:
```suggestion
    <string name="material_you_description">Use system colors across the app. Overrides static theme color option.</string>
```

File: src/android/app/src/main/java/io/github/mandarine3ds/mandarine/fragments/SetupFragment.kt [hunk 2]
```diff
@@ -370,6 +401,84 @@ class SetupFragment : Fragment() {
...
+            }
+            .setNegativeButton(android.R.string.cancel, null)
+            .show()
+    }
+
+    private fun showMaterialYouAndBlackThemeDialog() {
+        val preferences = PreferenceManager.getDefaultSharedPreferences(requireContext())
+        val switchContainer = LinearLayout(requireContext()).apply {
+            orientation = LinearLayout.VERTICAL
+            setPadding(64, 16, 64, 32)
```
Comment (Gamer64ytb) [inline src/android/app/src/main/java/io/github/mandarine3ds/mandarine/fragments/SetupFragment.kt:423]: Looks a bit "hacky", but seems to be needed to work, so i'll let it go like this for now.

**Review by Gamer64ytb — APPROVED** [review]
All seems fine now. Only needs some testing to see if all works fine and will be ready to merge.

**Comment by ghost** [comment]
> If you want to, you can make orange the default theme, i would reccomend doing so

What about this?

**Comment by Gamer64ytb** [comment]
> > If you want to, you can make orange the default theme, i would reccomend doing so
> 
> What about this?

I'll probably set as default, seems a nice idea

**Comment by Gamer64ytb** [comment]
During personal testing, I have noticed that when black backgrounds is toggled, it doesn't update colors while leaving dialog. Try to fix if possible

**Comment by ghost** [comment]
Confirmed the bug is due to the listener not being triggered if the setup is fixed

**Comment by Gamer64ytb** [comment]
All seems fine now. Thanks for this feature 🙂

**Comment by kleidis** [comment]
Confirmed the bug is due to the listener not being triggered if the setup is fixed
    Ay I'm the rizzler bro

**Pull request merged**